css/style.css

```diff
@@ -1487,9 +1487,23 @@ h2 a {
   max-width: 100%;
   margin: auto;
 }
-.blog-author {
+.blog-authors {
   font-style: italic;
 }
+.blog-author-link {
+  color: inherit;
+  margin-left: 0.5ch;
+  text-decoration: none;
+}
+.blog-author-link-label {
+  text-decoration: underline;
+}
+.blog-author-avatar {
+  border-radius: 50%;
+  display: inline;
+  vertical-align: middle;
+  width: 1.5em;

```

Comment:
```suggestion
  width: 2em;
  height: 2em;
```
32px avatar looks good

Reply:
I went with the smaller size because it matched up more with the existing line height so it didn't shift any elements vertically. With `2em` the image no longer aligns with the text, pushing the height of the post metadata to be larger than a regular paragraph. I think the post view looks a little janky with larger images. Happy to go with it if others disagree but would like more opinions.

![Screenshot 2025-05-28 at 14 20 11](https://github.com/user-attachments/assets/5b83b441-9c04-4045-85af-fbab4e806bff)

Reply:
> image no longer aligns with the text

most of the websites use not aligned image with the text. text always align at center with respect to the image. [example](https://developer.mozilla.org/en-US/blog/color-models-humans-devices/).